configuration/etl_ddl/hiv_care_enrolment/hiv_enrolment.sql

```diff
@@ -0,0 +1,99 @@
+DELIMITER //
```

Comment:
@Michaelndula please maintain this as a create table statement without the stored precedure

Reply:
@moshonk I removed the stored procedure

Reply:
@Michaelndula seems like you didn't push this update.